src/components/Loaders/cardShimmer/cardShimmer.module.scss

```diff
@@ -46,12 +52,6 @@
   }
 }
 
-.animate {
-  animation: shimmer 2s infinite linear;
-  background: linear-gradient(to right, #eff1f3 4%, #e2e2e2 25%, #eff1f3 36%);
-  background-size: 1000px 100%;
-}
-
 @keyframes shimmer {
```

Comment:
I guess @whyDontI  meant to say integrate this as well into the `.wrapper` block, just like `fullview` 👍🏾